docs/www/data-transform.md

```diff
@@ -0,0 +1,393 @@
+---
+title: Data transformation
+order: 0
+---
+# Data transformation
+
+The basic process of streaming data is to have systems produce events to a topic
+and have other systems consume from that topic.
+What if you need to change the content of the events, or transform the events, 
+in some way like scrubbing private data?
+Normally you'd have to set up another process to consume the events, transform them,
+and produce them to another topic that your consumers can consume from.
+That's extra overhead of code, integration, monitoring, and performance load that you'd like to avoid if possible.
+
+With Redpanda, you can setup a process in Node.js and it will do all of the consume-transform-produce cycle for you.
+
+We do all of this co-processing using a [WASM (WebAssembly)](https://developer.mozilla.org/en-US/docs/WebAssembly) engine.
+
+All you need to do is to provide:
+
+- A JavaScript code that transforms the data. 
+- The source topics where to consume from.
+- The destination topics where produced data will be stored. 
```

Comment:
Maybe this can be omitted. It might be confusing because it may seem like the user needs to provide up-front the list of destination topics, but that isn't necessary. They are created on demand by parsing the return value of the `apply()` function.

Reply:
```suggestion

All you need to do is to provide:

- A JavaScript code that transforms the data. 
- The source topics where to consume from.

The destination topics can be generated on demand when your transformation runs. 
```